src/com/lftechnology/task/jan07/FileUtils.java

```diff
@@ -0,0 +1,147 @@
+package com.lftechnology.task.jan07;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+/**
+ * This is a simple utility class file for handling io operations for the files.
+ * 
+ * @author Sanjib<sanjibmaharjan@lftechnology.com>
+ * 
+ */
+public class FileUtils {
+
+	public static final String PATH_SEPERATOR = "/";
+	public static final String DOUBLE_QUOTE = "\"";
+
+	public static void main(String[] args) {
+
+	}
+
+	/**
+	 * @param filePath
+	 *            complete path to the file with its filename
+	 * @return lists of words obtained by delimiting the contents of file by 'space' character.
+	 * @throws IOException
+	 */
+	public static ArrayList<String> readFile(String filePath) throws IOException {
+		File file = new File(filePath);
+		file.createNewFile();
+		// Get data from this file using a file reader.
+		FileReader fileReader = new FileReader(file);
+		// To store the contents read via File Reader
+		Scanner readFile = new Scanner(fileReader);
+		ArrayList<String> fileChunks = new ArrayList<String>();
```

Comment:
Same here, create instance from the base class or interface if available, Here
List<<String>> fileChunks = new ArrayList<<String>>();